vmas/scenarios/transport.py

```diff
@@ -381,6 +381,59 @@ def default_observation(self, agent: Agent):
                     rel_mass, device=self.world.device
                 ).repeat(self.world.batch_dim, 1),
             ]
+        return capability_repr
+
+    
+    def partial_observation(self, agent: Agent):
+        """
+        Parital observation of the boxes for the agent
+        """
+         # get positions of all entities in this agent's reference frame
+        package_obs = []
+        out_of_obs_val = -0.0001 # default value used for out-of-observation data in the observation vector
+        for i, package in enumerate(self.packages):
+            # box starting position and goal position alway part of the observation
+            package_obs.append(self.og_package_positions[i])
+            package_obs.append(package.on_goal.unsqueeze(-1))
+            
+            mask = (torch.linalg.vector_norm(package.state.pos - agent.state.pos, dim=-1) < self.package_observation_radius)
+            pkg_state_vec = package.state.pos.clone()
+            pkg_vel_vec = package.state.vel.clone()
+            pkg_dist_to_goal_vec = package.state.pos - package.goal.state.pos
+            agent_dist_to_pkg_vec = package.state.pos - agent.state.pos
+            
+            pkg_state_vec[~mask] = out_of_obs_val
```

Comment:
Add package rotation to partial observability function observation

Reply:
Done for both, see [commit](https://github.com/GT-STAR-Lab/VectorizedMultiAgentSimulator/pull/18/commits/23d205951a2aed89c7782afbaa5c1a952931a416)